src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2030,6 +2030,14 @@ From there, simply click on one of the supported files in the File Explorer and
     <value>Draw crosshairs centered around the mouse pointer.</value>
     <comment>Mouse as in the hardware peripheral.</comment>
   </data>
+  <data name="Oobe_MouseUtils_MouseJump.Text" xml:space="preserve">
+    <value>Mouse Jump</value>
+    <comment>Mouse as in the hardware peripheral.</comment>
+  </data>
+  <data name="Oobe_MouseUtils_MouseJump_Description.Text" xml:space="preserve">
+    <value>Teleport the mouse pointer quickly to any screen.</value>
```

Comment:
@jaimecbernardo - minor suggestion "<value>Jump the mouse pointer quickly to anywhere on your desktop.</value>"

Reply:
Sounds good to me.